feature/document/src/main/java/com/mifos/feature/document/document_dialog/DocumentDialogScreen.kt

```diff
@@ -0,0 +1,382 @@
+        }
+
+        return temp
+    }
+
+    Dialog(onDismissRequest = { setShowDialog(false) }) {
+        Surface(
+            shape = RoundedCornerShape(16.dp),
+            color = Color.White
+        ) {
+            Box(
+                contentAlignment = Alignment.Center
+            ) {
+                Column(modifier = Modifier.padding(20.dp)) {
+
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.SpaceBetween,
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Text(
+                            text = dialogTitle,
+                            fontSize = MaterialTheme.typography.titleLarge.fontSize,
+                            color = BluePrimary
+                        )
+                        Icon(
+                            imageVector = Icons.Filled.Cancel,
+                            contentDescription = "",
+                            tint = colorResource(android.R.color.darker_gray),
+                            modifier = Modifier
+                                .width(30.dp)
+                                .height(30.dp)
+                                .clickable { setShowDialog(false) }
+                        )
+                    }
+
+                    Spacer(modifier = Modifier.height(20.dp))
+
+                    MifosOutlinedTextField(
+                        value = name,
+                        onValueChange = { value ->
+                            name = value
+                            nameError = false
+                        },
+                        label = R.string.feature_document_name,
+                        error = if(nameError) R.string.feature_document_message_field_required else null,
+                        trailingIcon = {
+                            if (nameError) {
+                                Icon(imageVector = Icons.Filled.Error, contentDescription = null)
+                            }
+                        }
+                    )
+
+                    Spacer(modifier = Modifier.height(20.dp))
+
+                    MifosOutlinedTextField(
+                        value = description,
+                        onValueChange = { value ->
+                            description = value
+                            descriptionError = false
+                        },
+                        label = R.string.feature_document_description,
+                        error = if(descriptionError) R.string.feature_document_message_field_required else null,
+                        trailingIcon = {
+                            if (descriptionError) {
+                                Icon(imageVector = Icons.Filled.Error, contentDescription = null)
+                            }
+                        }
+                    )
+
+                    Spacer(modifier = Modifier.height(20.dp))
```

Comment:
The gap between name-description && description- selected file is a lot... can we reduce it?